src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,104 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.ToDo;
```

Comment:
I like that your import packages are well-separated! It follows the convention of using lowercase letters and separating words with dots.